lib/Service/ApiService.php

```diff
@@ -233,14 +252,14 @@ public function sync(Session $session, Document $document, $version = 0, $autosa
 		}
 
 		try {
-			$result['document'] = $this->documentService->autosave($document, $file, $version, $autosaveContent, $documentState, $force, $manualSave, $token, $this->request->getParam('filePath'));
-		} catch (DocumentSaveConflictException $e) {
+			$result['document'] = $this->documentService->autosave($document, $file, $version, $autosaveContent, $documentState, $force, $manualSave, null, $this->request->getParam('filePath'));
```

Comment:
I think you'd still want to hand over the `token` - now called `shareToken` as provided as an argument.

```suggestion
			$result['document'] = $this->documentService->autosave($document, $file, $version, $autosaveContent, $documentState, $force, $manualSave, $shareToken, $this->request->getParam('filePath'));
```

Reply:
Absolutely, good catch

Reply:
I applied the suggestion as a fixup to see if that helps with cypress.